code/socialDistribution/requests.py

```diff
@@ -34,6 +34,11 @@ def get(url, params=None):
         "Accept": "application/json"
     }
 
+    # add auth credentials if getting remote node data as some may require it
+    if basicAuthCredentials:
+        authToken = base64.b64encode(basicAuthCredentials).decode("ascii")
+        headers['Authorization'] = 'Basic %s' % authToken
+
```

Comment:
Can we set up this block to work in the same way as in the post() function? 

E.g.
```python
  # Add Basic Auth Header specific to a node for Inbox api
  host = url_parser.get_host(url)
  auth_credentials = node_manager.get_credentials(host=host, remote_credentials=True)
  if (sendBasicAuthHeader and auth_credentials):
      authToken = base64.b64encode(auth_credentials).decode("ascii")
      headers['Authorization'] = 'Basic %s' % authToken
```

I just think it would be easier to use if get() and post() have the same interface and we don't have to pass in the credentials into get() but not post()

Reply:
Also, can we add the auth code to the delete function as well? Just in case?

Reply:
For sure, thanks!